samples/bot-conversation-sso-quickstart/csharp_dotnetcore/BotConversationSsoQuickstart/Resources/ABSSORefresh.json

```diff
@@ -0,0 +1,35 @@
+﻿{
+  "type": "AdaptiveCard",
+  "refresh": {
+    "action": {
+      "type": "Action.Execute",
+      "verb": "initiateSSO",
+      "title": "RefreshTitle"
+    }
+  },
+  "authentication": {
+    "text": "Signin Text",
+    "connectionName": "newConnection",
+    "tokenExchangeResource": {
+      "id": "9b6dcf71-df47-4756-b486-47ca94dbacee"
+    },
+    "buttons": [
+      {
+        "type": "signin",
+        "title": "Please sign in",
+        "value": "https://token.botframework.com/api/oauth/signin?signin=9b6dcf71-df47-4756-b486-47ca94dbacee"
```

Comment:
Why did we push app id?  9b6dcf71-df47-4756-b486-47ca94dbacee

Add place holder and update the readme file.

Reply:
Changes updated